woocommerce/woocommerce-gutenberg-products-block
# Featured Items: Remove inline style defaults for color options

This replaces the inline style in the Featured Product and Featured Category blocks with an editor-specific styling that has a bit more specificity - but not too much to where it overrides the Site Editor styles.





Fixes #7042 




### Screenshots



| Before | After |
| ------ | ----- |
|  ![CleanShot 2022-09-01 at 14 31 05](https://user-images.githubusercontent.com/481776/187987970-e86aebfd-f7ae-4eda-a6fb-622ef53f5899.png)  |  ![CleanShot 2022-09-01 at 14 35 45](https://user-images.githubusercontent.com/481776/187988035-f413cc44-3f39-48bb-bf68-597bb5d81394.png)  |


### Testing



1. Ensure you have a block theme active (i.e., Twenty Twenty-Two).
2. Go to **Appearance > Editor (Beta)**.
3. Click on the **Global Styles** panel and select the **Blocks** section.
4. Scroll down to the **Featured Product** or **Featured Category** block and edit the text and the background Global Styles.
5. Confirm that both the text and background colors reflect the changes from the sidebar controls.


### WooCommerce Visibility




### Changelog

> Featured Item Blocks: remove inline default color so that custom colors from Global Styles are applied correctly.
